cluster/charts/rook/templates/psp.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.pspEnable }}
+# PSP for rook-operator
+# name starts with 00- helps in policy order to keep it at the top - https://kubernetes.io/docs/concepts/policy/pod-security-policy/#policy-order
```

Comment:
can you add more details to the comment about why the rook psp needs to be at the top?